modules/popup/react/lib/Popper.tsx

```diff
@@ -119,12 +116,12 @@ const OpenPopper = React.forwardRef<HTMLDivElement, PopperProps>(
       containerElement,
       ...elemProps
     },
-    forwardRef: React.RefObject<HTMLDivElement>
+    ref
   ) => {
     const firstRender = React.useRef(true);
     const popperInstance = React.useRef<PopperJS.Instance>();
     const [placement, setPlacement] = React.useState(popperPlacement);
-    const stackRef = usePopupStack(forwardRef, anchorElement as HTMLElement);
```

Comment:
Possible breaking change. Before, if `forwardRef` was defined, `stackRef` would be the same reference. Now they will always be different references since `usePopupStack` now uses forked refs. This is much more safe.